prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -395,6 +406,7 @@ class ReportFunction {
     ): String {
         val factory = JsonFactory()
         val outStream = ByteArrayOutputStream()
+        val itemRouting: List<ItemRouting> = createItemJurisdictionRouting(result, actionHistory)
```

Comment:
I dont full grasp the logic inside the function **createItemJurisdictionRouting**, but wouldn't be possible to make the execution of this call to occur only if VERBOSE_TRUE = true (on line 423) ?
The logic inside the function is not easy to follow up, maybe my requeist is not a valid one, but I'm asking just to be sure.